config/components/visibility/flowschema.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1
```

```suggestion
apiVersion: flowcontrol.apiserver.k8s.io/v1beta1
```

This should fix your errors.

---

Yes, it can. But is it ok to use Beta?

---

"flowcontrol.apiserver.k8s.io/v1beta3" is supported between 1.26 - 1.31.
And in v1.32, the version will be removed.

---

> Yes, it can. But is it ok to use Beta?

Is there any reasons why you want to avoid the v1beta3?

---

> "flowcontrol.apiserver.k8s.io/v1beta3" is supported between 1.26 - 1.31.
And in v1.32, the version will be removed.

SGTM

---

Now it's working fine. Thanks!

---

Additionally, could you open an issue to migrate v1beta3 to v1 once we drop the 1.28 supporting?

---

Sure.